pre-commit/check-incompatibilities.sh

```diff
@@ -9,8 +9,8 @@ fi
 
 MERGE_BRANCH="master"
 
-if [[ "${ORG:-}" == "dfinity-sandbox" ]]; then
-    MERGE_BRANCH="mirroring"
+if [[ "${REPO:-}" == "ic-private" ]]; then
+    MERGE_BRANCH="master-private"
 fi
```

Comment:
Shouldn't we set the MERGE_BRANCH to the merge base of the PR, i.e. [`GITHUB_BASE_REF`](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables)

Reply:
Ah yeah, that's a better solution 👍